apis/cloud.redhat.com/v1alpha1/clowdapp_types.go

```diff
@@ -419,31 +421,11 @@ func (i *ClowdApp) GetClowdSAName() string {
 	return fmt.Sprintf("%s-app", i.GetClowdName())
 }
 
-// ConvertToNewShim converts an old "pod" based spec into the new "deployment" style.
-func (i *ClowdApp) ConvertToNewShim() {
-	deps := []Deployment{}
-	for _, pod := range i.Spec.Pods {
-		dep := Deployment{
-			Name:        pod.Name,
-			Web:         pod.Web,
-			MinReplicas: pod.MinReplicas,
-			PodSpec: PodSpec{
-				Image:          pod.Image,
-				InitContainers: pod.InitContainers,
-				Command:        pod.Command,
-				Args:           pod.Args,
-				Env:            pod.Env,
-				Resources:      pod.Resources,
-				LivenessProbe:  pod.LivenessProbe,
-				ReadinessProbe: pod.ReadinessProbe,
-				Volumes:        pod.Volumes,
-				VolumeMounts:   pod.VolumeMounts,
-			},
-		}
-		deps = append(deps, dep)
-	}
-	i.Spec.Deployments = deps
-}
+// func (r *ClowdApp) SetupWebhookWithManager(mgr ctrl.Manager) error {
```

Comment:
Is this not required? It looks to be the boilerplate from the `kubebuilder webhook create`. Does this get covered by registering both webhooks in `run.go`?

Reply:
It seems like it does. I tried removing it and re-running - and everything worked out fine.